src/app/static/src/app/generated.d.ts

```diff
@@ -0,0 +1,38 @@
+/**
+  * This file was automatically generated.
+  * DO NOT MODIFY IT BY HAND.
+  * Instead, modify the hintr JSON schema files
```

Comment:
Is this now slightly out of date? Ran locally and saw a couple of differences in the Response types as well as the corrected hintr->mintr message.

Reply:
There are some bugs in the pkgapi schemas so I did in fact edit this by hand! But yes I'll also manually fix the hintr -> mintr

Reply:
(PR for fixing the pkgapi schemas is in progress at the moment https://github.com/reside-ic/pkgapi/pull/10)